client/src/components/SideNav.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+const Container = styled.nav`
+  color: ${p => p.theme.color.gray};
+  font-size: 1.15em;
+  line-height: 1.5em;
+  user-select: none;
+  width: 300px;
+  display: flex;
+  justify-content: center;
+  padding: 5rem 0;
+  font-family: 'Sen';
+  transition: 0.2s ease-in;
+  ul {
+    list-style: none;
+    position: fixed;
+  }
+  li {
+    padding: 0.5rem 0;
+  }
+  li:hover {
+    color: ${p => p.theme.color.primary};
+  }
+`;
+
+function SideNav() {
+  return (
+    <Container>
+      <ul>
+        <li>Basic Information</li>
+        <li>About Me</li>
+        <li>About Course</li>
```

Comment:
if this is a side bar for someones profile, will this appear on everyones profile? Or only people who are offering a course, and are we only allowing someone to offer one course?

Reply:
This is only on the edit profile page so this will only be for instructors. Eventually the idea is that the left navigation will navigate subsections on the current page and will include a link for course management, which will have its own subsections.